pkgs/desktops/kde-5/applications/srcs.nix

```diff
@@ -2,6 +2,14 @@
 { fetchurl, mirror }:
 
 {
+  kronometer =  {
```

Comment:
this script is autogenerated. You can give the `src` to `kdeApp`. See for e.g. `kde5.kile`, `kde5.konversation`, or `kde5.yakuake`.